packages/cli/src/tools/brewInstall.ts

```diff
@@ -6,15 +6,16 @@ import ora from 'ora';
 type InstallArgs = {
   pkg: string;
   label?: string;
+  succeedMessage?: string;
   loader: ora.Ora;
 };
 
-async function brewInstall({pkg, label, loader}: InstallArgs) {
+async function brewInstall({pkg, label, succeedMessage, loader}: InstallArgs) {
```

Comment:
Would this be cleaner if we passed `onSuccess` and `onError` handlers instead?

Reply:
Hmm, yes.

So, we call `onSuccess` only if it's not `undefined`, we default to showing `loader.succeed()`?